### PR TITLE
Reduced CHANGELOG for 2.4

### PR DESCRIPTION
**Pull request by daschuer**
This is now the reduced version as discussed. 
I have added a Track Properties and Preferences region.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -833,326 +627,39 @@
...
+* OpenBSD: Allow building Mixxx [#11083](https://github.com/mixxxdj/mixxx/pull/11083)
+* Improve Linux launcher
+  [#11826](https://github.com/mixxxdj/mixxx/pull/11826)
+  [#11820](https://github.com/mixxxdj/mixxx/issues/11820)
+  [#11805](https://github.com/mixxxdj/mixxx/pull/11805)
+  [#12424](https://github.com/mixxxdj/mixxx/pull/12424)
+
+### Track properties
+
+* Fixed a SIGSEGV after a debug assertion [#4316](<https://github.com/mixxxdj/mixxx/pull/4316>
```
Comment (ronso0) [inline CHANGELOG.md:649]:
```suggestion
* Fix a SIGSEGV after a debug assertion [#4316](<https://github.com/mixxxdj/mixxx/pull/4316>
```

**Comment by ronso0** [comment]
Thank you, just some comments.
Anyone else wantst to take a look?

Once this is merged I'll do more cleanup / renaming / consolidation.

**Comment by daschuer** [comment]
The build failure is unrelated and fixed here: https://github.com/mixxxdj/mixxx/pull/12581

**Comment by ronso0** [comment]
Yeah, thanks for the fix.

Unless someone objects I'll merge tomorrow morning.

**Comment by ronso0** [comment]
Okay, so let's go.

**Pull request merged**